source/compute/instance-types.rst

```diff
@@ -0,0 +1,247 @@
+    - Maximum Time
+  * - c2-burst.c1r05
+    - 5%
+    - 100%
+  * - c2-burst.c1r1
+    - 10%
+    - 100%
+  * - c2-burst.c1r2
+    - 20%
+    - 100%
+  * - c2-burst.c2r4
+    - 40%
+    - 200%
+  * - c2-burst.c2r8
+    - 60%
+    - 200%
+  * - c2-burst.c4r16
+    - 90%
+    - 400%
+  * - c2-burst.c8r32
+    - 135%
+    - 800%
+
+*****************
+Accelerated Types
+*****************
+
+These compute types feature additional hardware accelerators, for
+workloads that are written to specifically target the accelerator
+provided. They are not suitable as general purpose servers, as they
+are inherently not as cost efficient if the accelerator is not used.
+
+.. list-table:: Accelerated Types
+    :header-rows: 1
+    :widths: 10 18 18 18 18 18
+
+    * - Type
+      - Vendor/Arch
+      - Generation
+      - vCPU Policy
+      - pCPU Policy
+      - Accelerator
+    * - c1a-gpu
+      - AMD x86-64
+      - Milan
+      - Each vCPU is a thread of a pCPU
+      - pCPU are dedicated to a single VM
+      - 1 or more NVIDIA RTX A6000 48GB GPU
+    * - c2-gpu
+      - Intel x86-64
+      - Skylake
+      - Each vCPU is a thread of a pCPU
+      - pCPU are shared between VMs
+      - Slice of NVIDIA A100 80GB GPU
+    * - c3-gpu
+      - Intel x86-64
+      - Sapphire Rapids
+      - Each vCPU is a thread of a pCPU
+      - pCPU are dedicated to a single VM
+      - 1 or more NVIDIA L40S 48GB GPU
+
+It is important to note that your workloads and operating systems
+using accelerated types must have appropriate drivers and licenses
+for any accelerator present.
+
+Accelerated types may have additional requirements the OS must follow
+for the accelerator to be functional. For GPUs, see :ref:`gpu-support`.
+
+**********************
+vCPU and pCPU Policies
+**********************
+
+A "vCPU" means a CPU that the virtual machine sees as able to be used
+by the operating system, and a "pCPU" means a CPU core that is actually
+physically supporting the execution of the vCPU.
+
+Instance types may allocate a vCPU as a thread of a pCPU where the
+pCPU supports threads. Some pCPU do not support threads, and in those
+cases a vCPU is simply allocated to a pCPU.
+
+An instance type may also treat pCPU as shared between more than one
+VM, or dedicated to a single specific VM. Where the pCPU is dedicated
+and vCPUs are threads of a pCPU, the threads are always allocated as
+siblings on pCPUs.
+
+It should be noted that vCPU performance depends on whether the pCPU
+is threaded, and whether the pCPUs are shared. The exact behavior of
+threads on a pCPU is documented in the CPU vendor's documentation or
+datasheets.
+
+.. _change-instance-type:
+
+********************************
+Changing type of virtual servers
+********************************
+
+You can change the type of virtual server after it has been created.
+The instance type can be changed by performing an offline resize. The
+server must be shutdown, and then you can select a different type and
```

Comment:
The wording suggests that the instance must be shut-down before the user can resize, but the resize process will shut-down the server for the user.

Reply:
Heavily reworded all this to mostly point at resize page and note resize can shutdown server or you can do yourself.